go.mod

```
@@ -1,6 +1,6 @@
 module github.com/rook/kubectl-rook-ceph
 
-go 1.21
+go 1.22
```

```suggestion
go 1.22
```

The changes are good to me, only reason we have not done this yet is the `rook/rook` project still is on `1.21` 

@travisn should we plan to move to 1.22 in the main rook repo? I know the main repo is not directly related to this repo but just to be same page I put this questions

---

I'd prefer to wait a bit longer before requiring 1.22 in this or the rook repo, unless there is a need for the change. It's nice to support the two most recent minor versions of go.

---

yeah, that is fair. (I did not know the main repo coupling in here)